docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -9,12 +9,11 @@ systems like MySQL and Hive, or between two different MySQL instances.
 Requirements
 ------------
 
-Requirements for using the connector in a catalog to connect to a MySQL data
-source are:
+To connect to a MySQL data source using the connector in a catalog, you need:
 
-* MySQL 5.6, 5.7, 8.0 and higher
-* Network access, by default on port 3306, from the Trino coordinator and
-  workers to MySQL.
+* MySQL 5.7, 8.0 or higher.
```

Comment:
Why was `5.6` removed?

Reply:
Is it because `5.6` reached end-of-life in Feb 2021?

Reply:
According to https://en.wikipedia.org/wiki/MySQL#Release_history, MySQL 5.6 is past its end-of-support date.

Reply:
They stopped supporting it in Feb 2021.